Tests/conf.json

```diff
@@ -1396,6 +1396,7 @@
         "Symantec MSS": "No instance, probably not going to get it (issue 15513)",
         "EclecticIQ Platform": "Instance Issues",
         "Google Cloud Compute": "Can't test yet",
+        "Zscaler": "No license, will be unskipped once it is renewed",
```

Comment:
Did you open an issue for renewing the license? if so please mention the issue number in the comment